### DISCUSSION TITLE
Make Contributor's Checklist to checkbox

### DISCUSSION THREAD

**Discussion by avamingli** [apache/cloudberry#20] [Ideas / Feature Requests] [answered]
### Proposers

Make Contributor's Checklist to checkbox for convenience.
Then users can click button to select.

### Proposal Status

Under Discussion

### Abstract

Pull request template.

### Motivation

convenience for users.

### Contributor's Checklist
Here are some reminders before you submit the pull request:
- [ ] Document changes
- [ ] Communicate in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (list them if needed)
- [x] Add tests for the change
- [x] Pass `make installcheck`
- [x] Pass `make -C src/test installcheck-cbdb-parallel`

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by avamingli** [comment]
@tuhaihe

**Comment by tuhaihe** [comment, marked as answer]
Hi @avamingli, good ideas! I will submit a PR for this.

_ps, I will move this entry to the 'ideas/feature request' category, we can share ideas there. The `proposal` category should be used for discussions on major changes to Cloudberry Database codebase._

Thanks again!

**Comment by tuhaihe** [comment]
Hi @avamingli, I submitted one PR for your suggestion: https://github.com/cloudberrydb/cloudberrydb/pull/91, PTAL. Thanks!
